plugin.php

```diff
@@ -295,6 +295,18 @@ function rest_validate_request_arg( $value, $request, $param ) {
 			}
 		}
 
+		if ( 'array' === $args['type'] ) {
+			// A comma-separated array of IDs could be acceptable instead of an array.
```

Comment:
In JSON Schema, I believe the way to define an arrays type is like so 

```json
    "options": {
            "type": "array",
            "minItems": 1,
            "items": { "type": "string" },
            "uniqueItems": true
        },
```

See http://json-schema.org/example2.html